mongorover/MongoCollection.lua

```diff
@@ -265,6 +265,34 @@ local MongoCollection = {__mode="k"}
 		return MongoCursor(self, cursor_t)
 	end
 
+	---
+	-- Create an index on any field or combination of fields in the collection.
+	-- Example usage at @{indexing.lua}.
+	-- @tparam table of fields and their corresponding index types
+	-- @tparam table of options to be used when creating index
+	-- @return string index name on success
```

Comment:
This function won't return nil, it will just error out because of this [line](https://github.com/mongodb-labs/mongorover/pull/66/files#diff-b3084e8d014c2b0adf0231692fda4e23R713), right? This might have to change in behavior to whatever pymongo does.